ui/src/containers/AppsTableContainer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { sortable, SortByDirection } from '@patternfly/react-table';
+import AppsTable from '../components/AppsTable';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getApps } from '../actions/actions';
+
+class AppsTableContainer extends React.Component {
+  constructor (props) {
+    super(props);
+    this.sortBy = {};
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+  }
+  componentDidMount () {
+    this.props.getApps();
+  }
+  onRowClick (event, rowId, props) {
+    this.setState({ redirect: true });
+  }
+  onSort (_event, index, direction) {
```

Comment:
@craicoverflow will take a look at this one soon. I think I lost something in the Redux refactor. There is a separate [JIRA](https://issues.jboss.org/browse/AEROGEAR-8373) so may wait until this one is complete to revisit this. I'm conscious to not let the scope of this pr get too big.

Reply:
@laurafitzgerald +1, this can definitely be corrected in https://issues.jboss.org/browse/AEROGEAR-8373, where it would be best handled in a reducer function, and all state mgt taken out of the component and handled by redux